### PR TITLE
fix(agoric-cli): skip initial octothorp

### LINKED ISSUE #7702: SyntaxError#1: Unexpected token # in JSON at position 0: agops perf satisfaction not aware of smallcaps
4e5ca113512d831026c29a0c9d37703f93fef72b

```
root@7a7f0ec46caf:/usr/src/agoric-sdk# OFFER=$(mktemp -t agops.XXX)
root@7a7f0ec46caf:/usr/src/agoric-sdk# agops vaults open --wantMinted 5.00 --giveCollateral 9.0 >|"$OFFER"
running with options {
  offerId: 1683857574594,
  collateralBrand: 'ATOM',
  wantMinted: 5,
  giveCollateral: 9
}
root@7a7f0ec46caf:/usr/src/agoric-sdk# agops perf satisfaction --executeOffer $OFFER --from $GOV1ADDR --keyring-backend=test
{
  sharedOpts: { keyringBackend: 'test' },
  opts: {
    executeOffer: '/tmp/agops.N6z',
    from: 'agoric18rl8v6kgf09fq0y3sutsc3alttrjky3nl2mdqy'
  }
}
(SyntaxError#1)
SyntaxError#1: Unexpected token # in JSON at position 0

  at JSON.parse (<anonymous>)
  at Command.<anonymous> (packages/agoric-cli/src/commands/perf.js:56:30)
  at packages/agoric-cli/src/bin-agops.js:76:17

root@7a7f0ec46caf:/usr/src/agoric-sdk# cat /tmp/agops.N6z | jq .
{
  "body": "#{\"method\":\"executeOffer\",\"offer\":{\"id\":1683857574594,\"invitationSpec\":{\"callPipe\":[[\"getCollateralManager\",[\"$0.Alleged: BoardRemoteATOM brand\"]],[\"makeVaultInvitation\"]],\"instancePath\":[\"VaultFactory\"],\"source\":\"agoricContract\"},\"proposal\":{\"give\":{\"Collateral\":{\"brand\":\"$0\",\"value\":\"+9000000\"}},\"want\":{\"Minted\":{\"brand\":\"$1.Alleged: BoardRemoteIST brand\",\"value\":\"+5000000\"}}}}}",
  "slots": [
    "board02152",
    "board0257"
  ]
}
```

to reproduce:
```
1. `git clone git@github.com:Agoric/agoric-sdk.git && cd agoric-sdk && checkout ari/vaultstest`
2. `cd packages/deployment/upgrade-test`
3. `make local_sdk build` (once)
4. `make run`
5. run the following commands as a block
```
```
OFFER=$(mktemp -t agops.XXX)
agops vaults open --wantMinted 5.00 --giveCollateral 9.0 >|"$OFFER"
agops perf satisfaction --executeOffer $OFFER --from $GOV1ADDR --keyring-backend=test
```

### PR DESCRIPTION
**Pull request by erights**
Staged on [ari/vaultstest](https://github.com/Agoric/agoric-sdk/tree/ari/vaultstest) for now.

Fixes #7702 

Haven't yet followed @arirubinstein 's repro instructions. But the issue pointed at a line that has the reported problem in the obvious way, so I'm first just posting the obvious fix to see what happens.

### CODE REVIEW

**Review by arirubinstein — APPROVED** [review]

File: packages/agoric-cli/src/commands/perf.js [hunk 1]
```diff
@@ -53,7 +55,10 @@ export const makePerfCommand = logger => {
       const sharedOpts = perf.opts();
       logger.warn({ sharedOpts, opts });
       const payloadStr = fs.readFileSync(opts.executeOffer).toString();
-      const { offer } = JSON.parse(JSON.parse(payloadStr).body);
+      const innerBody = JSON.parse(payloadStr).body;
+      innerBody.startsWith('#') ||
+        Fail`expected smallcaps encoded body: ${innerBody}`;
```
Comment (dckc) [inline packages/agoric-cli/src/commands/perf.js:60]: this isn't backward compatible. is that on purpose?
any reason not to do something more like `.replace(/^#/, '')`?

Comment (turadg) [reply packages/agoric-cli/src/commands/perf.js:60]: +1 to backwards compatibility. but not not worth delaying merging this rn

**Review by turadg — APPROVED** [review]

File: packages/agoric-cli/src/commands/perf.js (hunk 1 above)
Comment (erights) [reply packages/agoric-cli/src/commands/perf.js:60]: I shoulda done that. But if this one works, I'll fix it correctly tomorrow. Leaving this conversation unresolved.

Comment (gibson042) [reply packages/agoric-cli/src/commands/perf.js:60]: #7713

**Pull request merged**